packages/buendia-backup/data/usr/share/buendia/tests/50-backup-to-external

```diff
@@ -0,0 +1,31 @@
+BUENDIA_TEST_BACKUP_TARGET=loop/backup.$(date +%Y-%m-%d)
+
+# Given a configured "external" device
+# When the backup cron job runs
+test_10_run_backup_cron () {
+    mount_loopback
+    execute_cron_right_now backup
+    # This fails, but maybe shouldn't be treated as a failure?
+    # 2019-08-21 10:09:31 18191> (start) buendia-restore /dev/loop7
+    # 2019-08-21 10:09:31 18191> /dev/loop7 is a block device; mounting...
+    # 2019-08-21 10:09:31 18191> mount: wrong fs type, bad option, bad superblock on /dev/loop7,
+    # 2019-08-21 10:09:31 18191>        missing codepage or helper program, or other error
+    # 2019-08-21 10:09:31 18191> umount: /mnt/buendia-restore.18200.loop7: not mounted
+    # 2019-08-21 10:09:31 18191> (end, status 32)
+    true
```

Comment:
[discuss] I was initially surprised that the test doesn't fail when the `backup` command fails.  Should we run tests with `-e` so that non-zero exit codes immediately abort the test?  That would be my default choice, but I haven't thought about this as much as you have so I'm open to considering reasons either way.  Lmk what you think.

Reply:
I actually think that `set -e` makes sense in principle. In practice I think it makes the tests hard to get right, because some of the underlying cron stuff hasn't been adequately debugged yet. In particular `40-backup-to-local` fails with `set -e` and I haven't quite figured out why.

@zestyping I can block the merge and spend the time on making this work, or we can merge the set of working tests and add `set -e` as a subsequent goal. Thoughts?

Reply:
@schuyler If it's not too hairy, how about doing `set -e` by default and then turning it off in specific tests like `40-backup-to-local` where it doesn't work, with a `// TODO` to fix it later?

In any case, don't block.  If that's hard, just leave a `// TODO` to use `set -e`.

Reply:
Good idea. I'll implement that

Reply:
Done